Build.ps1

```diff
@@ -61,6 +61,19 @@ if (($BuildStep -ieq "all") -Or ($BuildStep -ieq "sdk")) {
   }
 }
 
+if (($BuildStep -ieq "all") -Or ($BuildStep -ieq "DevSetupAgent")) {
+  foreach ($platform in $env:Build_Platform.Split(",")) {
+    foreach ($configuration in $env:Build_Configuration.Split(",")) {
+      if ($Platform -ieq "arm64") {
+        HyperVExtension\BuildDevSetupAgentHelper.ps1 -Platform $Platform -Configuration $configuration -VersionOfSDK $env:sdk_version -SDKNugetSource $SDKNugetSource -AzureBuildingBranch $AzureBuildingBranch -IsAzurePipelineBuild $IsAzurePipelineBuild -BypassWarning
+      }
+      else {

```

Comment:
This is a minor issue since we very rarely build more than 1 platform at a time.  If someone is builing both x86 and x64, this will build DevSetupAgent x86 twice.